src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV2Spec.scala

```diff
@@ -0,0 +1,406 @@
+package org.broadinstitute.dsde.workbench.sam.api
```

Comment:
This test suite was a bit of a pain to set up. I wanted to mock out a lot of stuff and the option I went with was Mockito. Mockito took a good amount of fiddling to get it working properly and without _too_ much boiler plate, but I'm not entirely sure how clear the `setupRoutesTest` function is to anyone except me, so let me know if you think this is an okay way of mocking these tests or if you'd prefer something else.